### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by Pijussad** [skal3mar/Spotify-Premium-apk#1396]
From today in playlists shows only songs on device

**Comment by skiliz26** [comment]
Mine doesnt work... Dont even play a selected song (just previews)

**Comment by MrNoHair** [comment]
It brokey ;v 
Only artist album previews worked for me. The songs were grayed out, whenever I'd play an album it would quickly rotate several songs on the bottom of the screen (names+artist names+album art) and that would be it

**Comment by Limanu11** [comment]
Confirmed, something on Google update broke all spotify versions. I have 2 mobiles, one with a old version of Android and stopped working as well. Seems something related to Google play and services update

**Comment by job1022** [comment]
Same thing here :(

**Comment by 0davila** [comment]
Yeah i've got the same problem, all my playlist songs cant be played, even the artists albuns.

**Comment by Deephead** [comment]
Same here 🫤🫤🫤🫤

**Comment by Dineshgranger** [comment]
Use the may 2024 and sept 2023 versions, it's working fine ,I'm currently using may 2024 since the latest one asks us to install some other apk too and doesn't seem to come from the same developer who modded the previous versions. Apk credits a 3rd party modder not the one who's here

**Comment by Dineshgranger** [comment]
It also seems like the github page is taken over by bots/hackers since the issues opened are not actual "issues" but just promotions for apps, apks and other mods, that's why i suspect the katest version is modded by our dev

**Comment by Deephead** [comment]
So there is a workaround because the problem is from Google and not from the app.

Connect to VPN and choose Usa / Asia server.

Go to your Spotify account via browser (only) and change the region from your profile to Usa / Japan etc and save it.. 

Then close the vpn and open again the app.

Voila!

**Comment by Dawood190** [comment]
@Deephead 
It Worked!! Thanks a lot 👍👍

**Comment by 0davila** [comment]
> So there is a workaround because the problem is from Google and not from the app.
> 
> Connect to VPN and choose Usa / Asia server.
> 
> Go to your Spotify account via browser (only) and change the region from your profile to Usa / Japan etc and save it..
> 
> Then close the vpn and open again the app.
> 
> Voila!

It works 🙏🏾
Many thanks @Deephead
